sim_pkg/DesMom.txt

```diff
@@ -1,44 +0,0 @@
-0.0015000000000000042 
```

Comment:
Please don't commit garbage. It adds 0 value to your repo and just introduces junk.